@xen-orchestra/backups/formatVmBackups.mjs

```diff
@@ -25,6 +25,8 @@ function formatVmBackup(backup) {
       name_description: backup.vm.name_description,
       name_label: backup.vm.name_label,
     },
+    differencingDisks:  Object.values(backup.isVhdDifferencing).filter(t=>t).length,
+    keyDisks:  Object.values(backup.isVhdDifferencing).filter(t=>!t).length,
```

Comment:
Do not mix VHD and XO terminology.

- `differencingVhds` / `dynamicVhds`
- `incrementalDisks` / `keyDisks`